packages/app/package.json

```diff
@@ -78,6 +78,7 @@
     "ember-tooltips": "^2.9.2",
     "ember-truth-helpers": "^2.0.0",
     "ember-uuid": "1.0.0",
+    "eslint-config-prettier": "^3.0.1",
```

Comment:
Does this need to be in every addon? Or could we just add it to the global package.json

Reply:
If we create a root level eslint config file, I think we just need to add it to the global package.json.